Parsers/InfobloxNIOS/InfobloxNIOS.txt

```diff
@@ -27,21 +27,14 @@
 //
 let RawData = Syslog 
     | where Computer in ("datasource") and Facility == "local6"
-    | extend Parser = extract_all(@"^([a-zA-Z]{3}\s\d{2}\s[0-9\:]+)\s(\S+)\s([a-z_]+)(\[(\d+)\]|\:)(\:\s)?(.*)", dynamic([1,2,3,4,5,6,7]), SyslogMessage)
+    | extend Parser = extract_all(@"^(\d{2}\-[a-zA-Z]{3}\-\d{4}\s[0-9\.\:]+)?\s?([a-zA-Z-_]+)(\s|\:)?(.*)", dynamic([1,2,3,4]), SyslogMessage)
```

Comment:
Can you check regex? For me, it is returning Type as May from the derived field from Parser.

Query - Testing on sample data from Github: 

```
let Syslog = (externaldata(Computer:string, Facility:string, SyslogMessage: string, ProcessName:string, Type:string)
[@"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Sample%20Data/Syslog/InfobloxNIOS.json"]
with (format="multijson"));
let RawData = Syslog 
    | where Computer in ("datasource") and Facility == "local6"
    | extend Parser = extract_all(@"^(\d{2}\-[a-zA-Z]{3}\-\d{4}\s[0-9\.\:]+)?\s?([a-zA-Z-_]+)(\s|\:)?(.*)", dynamic([1,2,3,4]), SyslogMessage)
    | mv-expand todynamic(Parser)
    | extend responseTime = todatetime(Parser[0]),
            Type = tostring(Parser[1]),
            RawData_subString = tostring(Parser[3])
    | project-away Parser;
RawData
```

Reply:
I have changed this field in the parser to Log_Type, this data should be reflective in the new sample data